### ISSUE TITLE
Tranquility and brassb on Crown of the North Winds doesn't work

### ISSUE THREAD

**Issue by Dartan82** [torilmud/issues#522]
Name             : Breath of the Brass Dragon
Alias            : brassb                   
Type             : Melee                    
Attack Type      : Auto                     
Target           : Offense, Character in room
Recharge         : 12 minutes 
Restrictions     : Garrote, Standing, Can See
Race Restrict (T): Type Dragon, Type Elemental, Type Outsider, Type Construct
-=-=-=-=-=-=-=-=-=-=--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Target Effect    : Add sleep for 2 minutes                            
                   Save versus breath to avoid  
Target Effect    : --> Change state to Sleeping                       
Target Effect    : --> Stop merciful attackers                        
Target Effect    : --> Stop fighting                                  

Name             : Tranquility              
Alias            : tq                       
Type             : Spell                    
Source           : Arcane                   
Prepare          : Study                    
Target           : Self, Character in room, NPC only
Activation Time  : 2 seconds 2 pulses 
Related Skill    : spellcast illusion
Restrictions     : Autofire, Garrote, Standing, Can Speak
-=-=-=-=-=-=-=-=-=-=--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Target Effect    : Stop fighting                                      Radius: All 
                   Save versus spell to avoid  
Self Effect      : Wait for 4 seconds       

Basically combat continues even with tog vic off.

**Comment by Lilithelle** [comment]
Tested, verified, repair in progress, hopefully with next version of Bahamut.

**Comment by shevarash** [comment]
The Bahamut item was fixed, but leaving this open for Tranquility.

**Comment by Dartan82** [comment]
bump
